setup.py

```diff
@@ -25,85 +25,116 @@
 import six
 
 if six.PY3:
-    exec(open('ipopt/version.py', encoding="utf-8").read())
+    exec(open("cyipopt/version.py", encoding="utf-8").read())
 else:
-    exec(open('ipopt/version.py').read())
+    exec(open("cyipopt/version.py").read())
 
-PACKAGE_NAME = 'ipopt'
+PACKAGE_NAME = "cyipopt"
+DEPRICATED_PACKAGE_NAME = "ipopt"
```

Comment:
Should be `DEPRECATED_PACKAGE_NAME` (spelling)

I’m looking forward to this change and having the PyPI package name match the repo and conda-forge name 😊

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/a79218efb5ad219fe7c42f99f51edfe3a56a13a6 and https://github.com/matthias-k/cyipopt/pull/75/commits/ee8f73cfa8ec645da876b82baf22561ce45e0175.